### PR TITLE
jtbryan page update

### PR DESCRIPTION
**Pull request by jaredbryan881**
I updated my education section, edited my bio, added a photo, and added links for github, researchgate, orcid, and my email.

### CODE REVIEW

File: content/authors/jtbryan/_index.md [hunk 1]
```diff
@@ -4,32 +4,38 @@ authors:
...
-#  link: https://twitter.com/GeorgeCushen
-#- icon: google-scholar
-#  icon_pack: ai
-#  link: https://scholar.google.co.uk/citations?user=sIwtMXoAAAAJ
-#- icon: github
-#  icon_pack: fab
-#  link: https://github.com/gcushen
+  link: 'mailto:jtbryan@mit.edu'
+- icon: orcid
+  inon_pack: fab
```
Comment (beridel) [inline content/authors/jtbryan/_index.md]: Mispelling `inon_pack` -> `icon_pack`

File: content/authors/jtbryan/_index.md [hunk 2]
```diff
@@ -4,32 +4,38 @@ authors:
...
+  icon_pack: fab
+  link: https://github.com/jaredbryan881
 superuser: false
 title: Jared Bryan
 user_groups:
 - Group members
 ---
 
-Jared Bryan is starting at MIT fall 2020.
+Jared Bryan is a first-year graduate student starting at MIT in fall 2020.
```
Comment (beridel) [inline content/authors/jtbryan/_index.md:41]: This'll be something you can update whenever you'd like, but this space can be used to write a blurb or paragraph(s) about what you're interested in. No need though for the moment to change it.

**Review by beridel — CHANGES_REQUESTED** [review]

File: content/authors/jtbryan/_index.md (hunk 1 above)
Comment (jaredbryan881) [reply content/authors/jtbryan/_index.md]: Updated

File: content/authors/jtbryan/_index.md (hunk 2 above)
Comment (jaredbryan881) [reply content/authors/jtbryan/_index.md:41]: I will add an interests paragraph a bit later, but I will submit a PR before the semester starts.

**Review by beridel — APPROVED** [review]

**Pull request merged**